redis/pool.go

```diff
@@ -162,6 +162,9 @@ type Pool struct {
 	// the pool does not close connections based on age.
 	MaxConnLifetime time.Duration
 
+	// Trace options
+	TraceOptions TraceOptions
```

Comment:
Let’s document this more to show that every span created in relation to a new connection from
this pool will use the defined TraceOptions. Perhaps:
// TraceOptions defines the attributes that will be added to every span
// that is created in relation to every connection from this pool.